upgradability/set_code_hash/transparent_voting/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "transparent_voting"
+version = "0.1.0"
+authors = ["[your_name] <[your_email]>"]
```

Comment:
hehe, no :stuck_out_tongue: please add us

applies to all other packages

Reply:
Hmm, what email should I put here?

Reply:
usually we specify just: `authors = ["Cardinal Cryptography"]`